2018-09-07 14:58:12.252 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/  
2018-09-07 14:58:12.252 +05:30 [Information] Request starting HTTP/1.1 DEBUG http://localhost:55306/  0
2018-09-07 14:58:12.422 +05:30 [Information] Request finished in 236.0682ms 200 
2018-09-07 14:58:12.766 +05:30 [Information] Request finished in 584.7738ms 404 
2018-09-07 14:58:26.295 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger  
2018-09-07 14:58:26.303 +05:30 [Information] Request finished in 7.4904ms 301 
2018-09-07 14:58:26.307 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/  
2018-09-07 14:58:26.335 +05:30 [Information] Request finished in 26.1379ms 200 text/html
2018-09-07 14:58:26.352 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui.css  
2018-09-07 14:58:26.353 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-bundle.js  
2018-09-07 14:58:26.383 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-standalone-preset.js  
2018-09-07 14:58:26.412 +05:30 [Information] Sending file. Request path: '"/swagger-ui.css"'. Physical path: '"N/A"'
2018-09-07 14:58:26.413 +05:30 [Information] Sending file. Request path: '"/swagger-ui-bundle.js"'. Physical path: '"N/A"'
2018-09-07 14:58:26.415 +05:30 [Information] Sending file. Request path: '"/swagger-ui-standalone-preset.js"'. Physical path: '"N/A"'
2018-09-07 14:58:26.505 +05:30 [Information] Request finished in 132.0393ms 200 text/css
2018-09-07 14:58:26.528 +05:30 [Information] Request finished in 147.5764ms 200 application/javascript
2018-09-07 14:58:26.639 +05:30 [Information] Request finished in 283.2359ms 200 application/javascript
2018-09-07 14:58:26.910 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/v1/swagger.json  
2018-09-07 14:58:26.936 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/favicon-32x32.png  
2018-09-07 14:58:26.939 +05:30 [Information] Sending file. Request path: '"/favicon-32x32.png"'. Physical path: '"N/A"'
2018-09-07 14:58:26.943 +05:30 [Information] Request finished in 6.966ms 200 image/png
2018-09-07 14:58:26.948 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/favicon-16x16.png  
2018-09-07 14:58:26.951 +05:30 [Information] Sending file. Request path: '"/favicon-16x16.png"'. Physical path: '"N/A"'
2018-09-07 14:58:26.956 +05:30 [Information] Request finished in 6.8623ms 200 image/png
2018-09-07 14:58:27.136 +05:30 [Information] Request finished in 225.6154ms 200 application/json
2018-09-07 14:58:52.986 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/contact/emailverification?Email=sheikh.imran%40nagarro.com  0
2018-09-07 14:58:53.176 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.ContactController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.EmailVerificationRequest"]) - ModelState is Valid
2018-09-07 14:59:08.282 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.ContactController.Post (OnSolve.Mobile.Web)" in 15189.7376ms
2018-09-07 14:59:08.352 +05:30 [Error] Connection id ""0HLGKKGTJ4JJ8"", Request id ""0HLGKKGTJ4JJ8:00000003"": An unhandled exception was thrown by the application.
System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AsyncPoco.Database.<OpenSharedConnectionAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AsyncPoco.Database.<QueryAsync>d__54`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AsyncPoco.Database.<FetchAsync>d__40`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at OnSolve.Mobile.Web.Services.ContactsService.<IsContactDuplicatedWithinAccount>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at OnSolve.Mobile.Web.Controllers.ContactController.<Post>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.IISIntegration.IISMiddleware.<Invoke>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.RequestServicesContainerMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Frame`1.<ProcessRequestsAsync>d__2.MoveNext()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2018-09-07 14:59:08.359 +05:30 [Information] Request finished in 15370.3867ms 500 
2018-09-07 15:03:13.926 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/  
2018-09-07 15:03:13.931 +05:30 [Information] Request finished in 7.6546ms 200 text/html
2018-09-07 15:03:14.031 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui.css  
2018-09-07 15:03:14.037 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-standalone-preset.js  
2018-09-07 15:03:14.042 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-bundle.js  
2018-09-07 15:03:14.073 +05:30 [Information] The file "/swagger-ui-bundle.js" was not modified
2018-09-07 15:03:14.074 +05:30 [Information] The file "/swagger-ui-standalone-preset.js" was not modified
2018-09-07 15:03:14.100 +05:30 [Information] The file "/swagger-ui.css" was not modified
2018-09-07 15:03:14.100 +05:30 [Information] Request finished in 62.2296ms 304 application/javascript
2018-09-07 15:03:14.111 +05:30 [Information] Request finished in 73.7868ms 304 text/css
2018-09-07 15:03:14.117 +05:30 [Information] Request finished in 38.8272ms 304 application/javascript
2018-09-07 15:03:14.490 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/v1/swagger.json  
2018-09-07 15:03:14.499 +05:30 [Information] Request finished in 10.0893ms 200 application/json
2018-09-07 15:03:31.718 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/contact/emailverification?Email=sheikh.imran%40nagarro.com  0
2018-09-07 15:03:31.725 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.ContactController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.EmailVerificationRequest"]) - ModelState is Valid
2018-09-07 15:05:55.910 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2018-09-07 15:05:55.919 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.ContactController.Post (OnSolve.Mobile.Web)" in 144195.9982ms
2018-09-07 15:05:55.923 +05:30 [Information] Request finished in 144205.4071ms 200 
2018-09-07 15:19:55.507 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/api/contact?Code=iM!~(vL(6SlLEO5VkY)UL%5BX~SNZtTd_eYA%60wmjVXJtNj%609B%5DbFsIKvHcZr%60M(L_J  
2018-09-07 15:19:55.513 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.ContactController.Get (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.ContactListRequest"]) - ModelState is Valid
2018-09-07 15:21:39.213 +05:30 [Information] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext".
2018-09-07 15:21:39.256 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.ContactController.Get (OnSolve.Mobile.Web)" in 103757.0116ms
2018-09-07 15:21:39.259 +05:30 [Information] Request finished in 103786.8221ms 200 application/json; charset=utf-8
2018-09-07 15:39:36.446 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/sheikh.imran@nagarro.com/jwt application/json-patch+json 38
2018-09-07 15:39:37.104 +05:30 [Information] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext".
2018-09-07 15:39:37.109 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.JwtController.Post (OnSolve.Mobile.Web)" in 655.4536ms
2018-09-07 15:39:37.119 +05:30 [Information] Request finished in 678.5827ms 400 application/json; charset=utf-8
2018-09-07 15:52:08.923 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/  
2018-09-07 15:52:09.187 +05:30 [Information] Request finished in 286.0719ms 404 
2018-09-07 17:14:04.222 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/  
2018-09-07 17:14:04.222 +05:30 [Information] Request starting HTTP/1.1 DEBUG http://localhost:55306/  0
2018-09-07 17:14:04.222 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/  
2018-09-07 17:14:04.377 +05:30 [Information] Request finished in 233.3099ms 200 
2018-09-07 17:14:04.400 +05:30 [Information] Request finished in 259.5342ms 200 text/html
2018-09-07 17:14:04.447 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui.css  
2018-09-07 17:14:04.464 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-bundle.js  
2018-09-07 17:14:04.483 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-standalone-preset.js  
2018-09-07 17:14:04.526 +05:30 [Information] The file "/swagger-ui-bundle.js" was not modified
2018-09-07 17:14:04.528 +05:30 [Information] The file "/swagger-ui.css" was not modified
2018-09-07 17:14:04.530 +05:30 [Information] The file "/swagger-ui-standalone-preset.js" was not modified
2018-09-07 17:14:04.571 +05:30 [Information] Request finished in 107.9779ms 304 application/javascript
2018-09-07 17:14:04.571 +05:30 [Information] Request finished in 108.7297ms 304 application/javascript
2018-09-07 17:14:04.579 +05:30 [Information] Request finished in 121.776ms 304 text/css
2018-09-07 17:14:04.740 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/v1/swagger.json  
2018-09-07 17:14:04.976 +05:30 [Information] Request finished in 814.4972ms 404 
2018-09-07 17:14:05.132 +05:30 [Information] Request finished in 390.26ms 200 application/json
2018-09-07 17:16:03.908 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user application/json-patch+json 133
2018-09-07 17:16:04.551 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.UserRequest"]) - ModelState is Valid
2018-09-07 17:16:09.482 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 400
2018-09-07 17:16:09.494 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" in 5477.7099ms
2018-09-07 17:16:09.500 +05:30 [Information] Request finished in 5593.0189ms 400 
2018-09-07 17:16:19.219 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user application/json-patch+json 143
2018-09-07 17:16:19.225 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.UserRequest"]) - ModelState is Valid
2018-09-07 17:16:52.321 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" in 33097.3566ms
2018-09-07 17:16:52.415 +05:30 [Error] Connection id ""0HLGKMSR24B6R"", Request id ""0HLGKMSR24B6R:00000002"": An unhandled exception was thrown by the application.
System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AsyncPoco.Database.<OpenSharedConnectionAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AsyncPoco.Database.<ExecuteScalarAsync>d__38`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at OnSolve.Mobile.Web.Services.ContactsService.<VerifyRecipientAccount>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at OnSolve.Mobile.Web.Services.UserService.<AreRegistrationDetailsValid>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at OnSolve.Mobile.Web.Controllers.UserController.<Post>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.IISIntegration.IISMiddleware.<Invoke>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.RequestServicesContainerMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Frame`1.<ProcessRequestsAsync>d__2.MoveNext()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2018-09-07 17:16:52.419 +05:30 [Information] Request finished in 33199.6226ms 500 
2018-09-07 17:17:04.584 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user application/json-patch+json 143
2018-09-07 17:17:04.587 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.UserRequest"]) - ModelState is Valid
2018-09-07 17:17:15.747 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 400
2018-09-07 17:17:15.750 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" in 11163.5537ms
2018-09-07 17:17:15.754 +05:30 [Information] Request finished in 11169.5475ms 400 
2018-09-07 17:17:17.703 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user application/json-patch+json 143
2018-09-07 17:17:17.705 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.UserRequest"]) - ModelState is Valid
2018-09-07 17:24:51.044 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 400
2018-09-07 17:24:51.047 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" in 453342.0515ms
2018-09-07 17:24:51.050 +05:30 [Information] Request finished in 453348.6422ms 400 
2018-09-07 17:25:34.350 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/ens application/json-patch+json 59
2018-09-07 17:25:34.357 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.ENSLoginModel"]) - ModelState is Valid
2018-09-07 17:27:15.935 +05:30 [Information] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext".
2018-09-07 17:27:18.019 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" in 103662.8011ms
2018-09-07 17:27:18.022 +05:30 [Information] Request finished in 103672.4934ms 200 application/json; charset=utf-8
2018-09-07 17:51:44.749 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/contact/emailverification?Email=rajeev.soni%40nagarro.com  0
2018-09-07 17:51:44.785 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.ContactController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.EmailVerificationRequest"]) - ModelState is Valid
2018-09-07 17:51:53.534 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2018-09-07 17:51:53.536 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.ContactController.Post (OnSolve.Mobile.Web)" in 8773.2947ms
2018-09-07 17:51:53.540 +05:30 [Information] Request finished in 8790.2096ms 200 
2018-09-07 18:25:48.260 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/  
2018-09-07 18:25:48.260 +05:30 [Information] Request starting HTTP/1.1 DEBUG http://localhost:55306/  0
2018-09-07 18:25:48.381 +05:30 [Information] Request finished in 158.4428ms 200 
2018-09-07 18:25:48.750 +05:30 [Information] Request finished in 529.8134ms 404 
2018-09-07 18:25:55.301 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/  
2018-09-07 18:25:55.369 +05:30 [Information] Request finished in 25.9407ms 200 text/html
2018-09-07 18:25:55.372 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-standalone-preset.js  
2018-09-07 18:25:55.374 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui.css  
2018-09-07 18:25:55.376 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-bundle.js  
2018-09-07 18:25:55.467 +05:30 [Information] The file "/swagger-ui-bundle.js" was not modified
2018-09-07 18:25:55.467 +05:30 [Information] The file "/swagger-ui.css" was not modified
2018-09-07 18:25:55.469 +05:30 [Information] The file "/swagger-ui-standalone-preset.js" was not modified
2018-09-07 18:25:55.497 +05:30 [Information] Request finished in 121.5648ms 304 application/javascript
2018-09-07 18:25:55.502 +05:30 [Information] Request finished in 96.3645ms 304 application/javascript
2018-09-07 18:25:55.507 +05:30 [Information] Request finished in 124.2787ms 304 text/css
2018-09-07 18:25:55.658 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/v1/swagger.json  
2018-09-07 18:25:55.872 +05:30 [Information] Request finished in 212.3676ms 200 application/json
2018-09-07 18:26:53.195 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/api/contact?Code=!XN6wV8l(pJN%7DcO0W%5E6HWXA658N%5ELI4l%5D6LxuKDS3Z0F%7CXC%5Dh%7C%5EKf)*_WMu%5Ev%7Dvs  
2018-09-07 18:26:53.376 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.ContactController.Get (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.ContactListRequest"]) - ModelState is Valid
2018-09-07 18:27:44.381 +05:30 [Information] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext".
2018-09-07 18:27:44.416 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.ContactController.Get (OnSolve.Mobile.Web)" in 51116.8682ms
2018-09-07 18:27:44.418 +05:30 [Information] Request finished in 51225.4587ms 200 application/json; charset=utf-8
2018-09-07 18:30:16.169 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user application/json-patch+json 142
2018-09-07 18:30:16.810 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" with arguments (["OnSolve.Mobile.Web.Models.UserRequest"]) - ModelState is Valid
2018-09-07 18:30:46.284 +05:30 [Information] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext".
2018-09-07 18:30:46.290 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.UserController.Post (OnSolve.Mobile.Web)" in 30111.9194ms
2018-09-07 18:30:46.295 +05:30 [Information] Request finished in 30127.3777ms 200 application/json; charset=utf-8
2018-09-07 18:31:13.249 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/  
2018-09-07 18:31:13.254 +05:30 [Information] Request finished in 5.586ms 200 text/html
2018-09-07 18:31:13.278 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui.css  
2018-09-07 18:31:13.306 +05:30 [Information] The file "/swagger-ui.css" was not modified
2018-09-07 18:31:13.328 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-standalone-preset.js  
2018-09-07 18:31:13.350 +05:30 [Information] Request finished in 33.894ms 304 text/css
2018-09-07 18:31:13.354 +05:30 [Information] The file "/swagger-ui-standalone-preset.js" was not modified
2018-09-07 18:31:13.358 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/swagger-ui-bundle.js  
2018-09-07 18:31:13.359 +05:30 [Information] Request finished in 75.2753ms 304 application/javascript
2018-09-07 18:31:13.362 +05:30 [Information] The file "/swagger-ui-bundle.js" was not modified
2018-09-07 18:31:13.366 +05:30 [Information] Request finished in 83.1508ms 304 application/javascript
2018-09-07 18:31:13.472 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/v1/swagger.json  
2018-09-07 18:31:13.480 +05:30 [Information] Request finished in 8.2697ms 200 application/json
2018-09-07 18:31:13.491 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/favicon-32x32.png  
2018-09-07 18:31:13.498 +05:30 [Information] Sending file. Request path: '"/favicon-32x32.png"'. Physical path: '"N/A"'
2018-09-07 18:31:13.510 +05:30 [Information] Request finished in 18.8116ms 200 image/png
2018-09-07 18:31:13.610 +05:30 [Information] Request starting HTTP/1.1 GET http://localhost:55306/swagger/favicon-16x16.png  
2018-09-07 18:31:13.612 +05:30 [Information] The file "/favicon-16x16.png" was not modified
2018-09-07 18:31:13.615 +05:30 [Information] Request finished in 5.659ms 304 image/png
2018-09-07 18:32:55.221 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/rajeev.soni@nagarro.com/passwordresetchallenge  0
2018-09-07 18:32:55.260 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.PasswordController.Post (OnSolve.Mobile.Web)" with arguments (["rajeev.soni@nagarro.com"]) - ModelState is Valid
2018-09-07 18:32:55.907 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Post (OnSolve.Mobile.Web)" in 678.9679ms
2018-09-07 18:32:56.094 +05:30 [Error] Connection id ""0HLGKO4TP5445"", Request id ""0HLGKO4TP5445:00000001"": An unhandled exception was thrown by the application.
System.NullReferenceException: Object reference not set to an instance of an object.
   at OnSolve.Mobile.Web.Controllers.PasswordController.<Post>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.IISIntegration.IISMiddleware.<Invoke>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.RequestServicesContainerMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Frame`1.<ProcessRequestsAsync>d__2.MoveNext()
2018-09-07 18:32:56.098 +05:30 [Information] Request finished in 879.2258ms 500 
2018-09-07 18:33:24.887 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/rajeev.soni@nagarro.com/passwordresetchallenge  0
2018-09-07 18:33:24.896 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.PasswordController.Post (OnSolve.Mobile.Web)" with arguments (["rajeev.soni@nagarro.com"]) - ModelState is Valid
2018-09-07 18:36:44.064 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Post (OnSolve.Mobile.Web)" in 199141.0975ms
2018-09-07 18:36:44.085 +05:30 [Error] Connection id ""0HLGKO4TP5445"", Request id ""0HLGKO4TP5445:00000002"": An unhandled exception was thrown by the application.
System.NullReferenceException: Object reference not set to an instance of an object.
   at OnSolve.Mobile.Web.Controllers.PasswordController.<Post>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.IISIntegration.IISMiddleware.<Invoke>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.RequestServicesContainerMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Frame`1.<ProcessRequestsAsync>d__2.MoveNext()
2018-09-07 18:36:44.095 +05:30 [Information] Request finished in 199203.6668ms 500 
2018-09-07 18:36:46.142 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/rajeev.soni@nagarro.com/passwordresetchallenge  0
2018-09-07 18:36:46.147 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.PasswordController.Post (OnSolve.Mobile.Web)" with arguments (["rajeev.soni@nagarro.com"]) - ModelState is Valid
2018-09-07 18:37:54.606 +05:30 [Information] PasswordRestLink to user rajeev soni email address : rajeev.soni@nagarro.com is : successful
2018-09-07 18:37:57.300 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2018-09-07 18:37:57.305 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Post (OnSolve.Mobile.Web)" in 71160.3358ms
2018-09-07 18:37:57.309 +05:30 [Information] Request finished in 71168.516ms 200 
2018-09-07 18:38:58.450 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/password/6ZF$i4llYa5V%5Ey%7BIENmp application/json-patch+json 25
2018-09-07 18:38:58.468 +05:30 [Information] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext".
2018-09-07 18:38:58.477 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Create (OnSolve.Mobile.Web)" in 18.3126ms
2018-09-07 18:38:58.482 +05:30 [Information] Request finished in 31.4285ms 400 application/json; charset=utf-8
2018-09-07 18:39:13.209 +05:30 [Information] Request starting HTTP/1.1 POST http://localhost:55306/api/user/password/6ZF$i4llYa5V%5Ey%7BIENmp application/json-patch+json 29
2018-09-07 18:39:13.214 +05:30 [Information] Executing action method "OnSolve.Mobile.Web.Controllers.PasswordController.Create (OnSolve.Mobile.Web)" with arguments (["6ZF$i4llYa5V^y{IENmp", "OnSolve.Mobile.Web.Models.PasswordModel"]) - ModelState is Valid
2018-09-07 18:39:13.320 +05:30 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2018-09-07 18:39:13.324 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Create (OnSolve.Mobile.Web)" in 110.229ms
2018-09-07 18:39:13.328 +05:30 [Information] Request finished in 119.7604ms 200 
2018-09-07 18:40:06.462 +05:30 [Information] Request starting HTTP/1.1 PUT http://localhost:55306/api/user/rajeev.soni@nagarro.com/password application/json-patch+json 98
2018-09-07 18:40:06.489 +05:30 [Information] Authorization failed for user: null.
2018-09-07 18:40:06.495 +05:30 [Information] Authorization failed for the request at filter '"Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter"'.
2018-09-07 18:40:06.500 +05:30 [Information] Executing ChallengeResult with authentication schemes ([]).
2018-09-07 18:40:06.514 +05:30 [Information] AuthenticationScheme: "Bearer" was challenged.
2018-09-07 18:40:06.516 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Put (OnSolve.Mobile.Web)" in 48.9807ms
2018-09-07 18:40:06.519 +05:30 [Information] Request finished in 57.4536ms 401 
2018-09-07 18:40:34.173 +05:30 [Information] Request starting HTTP/1.1 PUT http://localhost:55306/api/user/rajeev.soni@nagarro.com/password application/json-patch+json 98
2018-09-07 18:40:34.176 +05:30 [Information] Authorization failed for user: null.
2018-09-07 18:40:34.180 +05:30 [Information] Authorization failed for the request at filter '"Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter"'.
2018-09-07 18:40:34.183 +05:30 [Information] Executing ChallengeResult with authentication schemes ([]).
2018-09-07 18:40:34.186 +05:30 [Information] AuthenticationScheme: "Bearer" was challenged.
2018-09-07 18:40:34.190 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Put (OnSolve.Mobile.Web)" in 12.9647ms
2018-09-07 18:40:34.216 +05:30 [Information] Request finished in 21.4354ms 401 
2018-09-07 18:40:35.439 +05:30 [Information] Request starting HTTP/1.1 PUT http://localhost:55306/api/user/rajeev.soni@nagarro.com/password application/json-patch+json 98
2018-09-07 18:40:35.444 +05:30 [Information] Authorization failed for user: null.
2018-09-07 18:40:35.469 +05:30 [Information] Authorization failed for the request at filter '"Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter"'.
2018-09-07 18:40:35.472 +05:30 [Information] Executing ChallengeResult with authentication schemes ([]).
2018-09-07 18:40:35.502 +05:30 [Information] AuthenticationScheme: "Bearer" was challenged.
2018-09-07 18:40:35.508 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Put (OnSolve.Mobile.Web)" in 62.3462ms
2018-09-07 18:40:35.511 +05:30 [Information] Request finished in 75.7273ms 401 
2018-09-07 18:41:01.575 +05:30 [Information] Request starting HTTP/1.1 PUT http://localhost:55306/api/user/rajeev.soni@nagarro.com/password application/json-patch+json 98
2018-09-07 18:41:01.577 +05:30 [Information] Authorization failed for user: null.
2018-09-07 18:41:01.580 +05:30 [Information] Authorization failed for the request at filter '"Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter"'.
2018-09-07 18:41:01.582 +05:30 [Information] Executing ChallengeResult with authentication schemes ([]).
2018-09-07 18:41:01.585 +05:30 [Information] AuthenticationScheme: "Bearer" was challenged.
2018-09-07 18:41:01.588 +05:30 [Information] Executed action "OnSolve.Mobile.Web.Controllers.PasswordController.Put (OnSolve.Mobile.Web)" in 9.864ms
2018-09-07 18:41:01.592 +05:30 [Information] Request finished in 16.2975ms 401 
